nixos/lib/make-options-doc/default.nix

```diff
@@ -126,11 +126,37 @@ let
     }
   '';
 
+  singleMdDoc = name: value: ''
+    ## ${lib.escape [ "<" ">" ] name}
+    ${value.description}
+
+    ${lib.optionalString (value ? type) ''
+      *_Type_*:
+      ${value.type}
+    ''}
+
+    ${lib.optionalString (value ? default) ''
+      *_Default_*
+      ```
+      ${builtins.toJSON value.default}
+      ```
+    ''}
+
+    ${lib.optionalString (value ? example) ''
+      *_Example_*
+      ```
+      ${builtins.toJSON value.example}
+      ```
+    ''}
+  '';
+
 in {
   inherit optionsNix;
 
   optionsAsciiDoc = lib.concatStringsSep "\n" (lib.mapAttrsToList singleAsciiDoc optionsNix);
 
+  optionsMdDoc = lib.concatStringsSep "\n" (lib.mapAttrsToList singleMdDoc optionsNix);
```

Comment:
MD should be capital as it's an abbreviation

Reply:
makes sense, fixed